package.json

```diff
@@ -15,6 +15,7 @@
     "test:record": "SUPERFACE_LIVE_API='*' dotenv -e .env -e .env.example -- jest",
     "check": "superface check",
     "lint": "superface lint",
+    "prepare": "superface prepare --station",
```

Comment:
I just realised that this will not work. The prepare command format is `superface prepare:<COMMAND>`

Reply:
So maybe: `"prepare:profile": "superface prepare:profile --station",`?